airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -268,11 +296,9 @@ def _check_success_task(self) -> None:
         self.log.info('ECS Task stopped, check status: %s', response)
 
         # Get logs from CloudWatch if the awslogs log driver was used
-        if self.awslogs_group and self.awslogs_stream_prefix:
-            self.log.info('ECS Task logs output:')
-            task_id = self.arn.split("/")[-1]
-            stream_name = f"{self.awslogs_stream_prefix}/{task_id}"
-            for event in self.get_logs_hook().get_log_events(self.awslogs_group, stream_name):
+        log_events = self._cloudwatch_log_events()
+        if log_events is not None:
+            for event in log_events:
```

Comment:
Woudn't it make more sense to return an empty list so you don't have to check if not none? So the loop will go over 0 items instead?

Or just do `log_events = self._cloudwatch_log_events() or []` (but I would suggest to do this in the function)

WDYT?

Reply:
Changed - great point, thanks.